### ISSUE TITLE
Failing to build from source

### ISSUE THREAD

**Issue by antiheld333** [Tinkerforge/brickd#15]
Hello, 
during `make install` i am getting the following error 

```
#: make install
libraries/tools:
- libusb:                1.0.21
- libudev:               not required
- pm-utils:              1.4.1
- systemd:               no
features:
- logging:               yes
- epoll:                 yes
- packet-trace:          no
- debug:                 no
- gprof:                 no
- red-brick:             no
- bricklet:              no
- mesh-single-root-node: no
- hotplug:               libusb
options:
- CFLAGS:                -std=gnu99 -Wall -Wextra -I.. -O2 -DSYSCONFDIR="\"/etc\"" -DLOCALSTATEDIR="\"/var\"" -I/usr/include/libusb-1.0 -DDAEMONLIB_WITH_LOGGING -DDAEMONLIB_WITH_EPOLL -DBRICKD_WITH_USB_REOPEN_ON_SIGUSR1
- LDFLAGS:               -pthread
- LIBS:                  -lrt -lusb-1.0 -ldl
MD /usr/bin
MD /etc
MD /etc/init.d
MD /etc/logrotate.d
MD /var/log
MD /var/run
MD /usr/share
MD /usr/share/man
MD /usr/share/man/man8
MD /usr/share/man/man5
CP brickd
CP brickd.conf
/bin/sh: 1: Syntax error: "(" unexpected
Makefile:569: recipe for target 'install' failed
make: *** [install] Error 2
failed to compile

```

I am using a Docker-Container to build the brickd with the following settings:

```
FROM ubuntu:latest

# Update Repos
RUN apt-get update

# Install required Packages
RUN apt-get install -y build-essential pkg-config libusb-1.0-0-dev libudev-dev pm-utils git

# Clone Main Element
RUN git clone https://github.com/Tinkerforge/brickd.git

WORKDIR /brickd/src

# Clone Sub-Repo
RUN git clone https://github.com/Tinkerforge/daemonlib.git

# Set Dir
WORKDIR /brickd/src/brickd

# Make and Install
RUN make
RUN make install
```

**Comment by photron** [comment]
I've fixed the syntax error in commit 17812d5f4a230edf47664ac339fed0bef667d212. Please try again with the current HEAD.

Thanks for reporting the problem.

**Comment by antiheld333** [comment]
Thanks. It works :) 
Issue can be closed
